File: sukjoo/Programmers/Level_2/Lv2_피보나치_수.java
class Solution {
    public int solution(int n) {
        int answer[] = new int[n+1];
        
        for (int i = 0; i <= n; i++){
            if(i==0) answer[i] = 0; // F(0) = 0
            else if (i==1) answer[i] = 1; // F(1) = 1
            else{
                int sum = answer[i-2] + answer[i-1]; //피보나치...
                answer[i] = sum % 1234567;
            }
        }
        return answer[n];
    }
}

File: hyunji/Programmers/Level_0/lv2_피보나치_수.py
def solution(n):
    fibolist = [0, 1]
    for i in range(2, n + 1):
        fibo = fibolist[i - 1] + fibolist[i - 2]
        fibolist.append(fibo)
    answer = fibolist[n]
    return answer % 1234567